tests/fields/test_fk.py

```diff
@@ -151,6 +156,11 @@ async def test_minimal__instantiated_create(self):
         tour = await testmodels.Tournament.create(name="Team1")
         await testmodels.MinRelation.create(tournament=tour)
 
+    async def test_minimal__instantiated_create_wrong_type(self):
+        author = await testmodels.Author.create(name="Author1")
+        with self.assertRaises(FieldError):
```

Comment:
You can use `assertRaisesRegex` to verify that it's failing because of the relation type.